app/src/common/components/Button.tsx

```diff
@@ -50,17 +51,30 @@ export const EditedButton = withStyles((theme) => ({
 type ButtonProps<C extends React.ElementType> = MuiButtonProps<
   C,
   { component?: C }
->;
+> & {
+  typographyColor?:
+    | "initial"
+    | "inherit"
+    | "primary"
+    | "secondary"
+    | "textPrimary"
+    | "textSecondary"
+    | "error";
+};
```

Comment:
I tried to extract the type of TypographyProps.color like this : 

```suggestion
  typographyColor?: Pick<TypographyProps, "color">;
```

This should work in my opinion.. But typescript keeps raising an error and I can't find out why :(

@tevariou If you have an idea i'm all ears... But in the meantime I guess we'll have to keep it like this.

Reply:
you can use `  typographyColor?: Pick<TypographyProps, "color">` but l.74 it should be `<Typography color={typographyColor.color}>{children}</Typography>` instead of `<Typography color={typographyColor}>{children}</Typography>`